elearning-backend/app/Http/Controllers/API/AdminWordsController.php

```diff
@@ -4,18 +4,20 @@
 
 use App\Http\Controllers\Controller;
 use App\Http\Requests\CreateWordsPostRequest;
+use App\Models\Category;
 use App\Models\Word;
 
 class AdminWordsController extends Controller
 {
-    public function index($id = null)
+    public function index($param = null)
```

Comment:
i dont know what you want to do here. Is it showing words list and specific word?
If so, you can separate the words list function as `index` then showing specific word as  `show` function.
I already shared the resources before, the laravel's resource functions